tutorials/kafka-exactly-once/src/main/resources/META-INF/properties.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0"?>
+<configuration>
+
+  <property>
+    <name>dt.application.KafkaExactlyOnce.operator.*.attr.MEMORY_MB</name>
+    <value>512</value>
+  </property>
+
+  <!-- set maxTuplesTotal for both BatchSequenceGenerator and ValidationToFile -->
+  <property>
+    <name>dt.application.KafkaExactlyOnce.operator.(sequenceGenerator)|(validationToFile).prop.maxTuplesTotal</name>
+    <value>20</value>
+  </property>
+  <!-- max tuples per window -->
+  <property>
+    <name>dt.application.KafkaExactlyOnce.operator.sequenceGenerator.prop.maxTuples</name>
+    <value>5</value>
+  </property>
+  <property>
+    <name>dt.application.KafkaExactlyOnce.operator.passthrough.prop.directoryPath</name>
+    <value>/tmp/kafka_exactly</value>
+  </property>
+  <property>
+    <name>dt.application.KafkaExactlyOnce.operator.passthrough.prop.tuplesUntilKill</name>
+    <value>5</value>
+  </property>
+
+  <!-- Do not change any topics for this application-->
+  <!-- KafkaExactlyOnceOutputOperator -->
+  <property>
+    <name>dt.application.KafkaExactlyOnce.operator.kafkaExactlyOnceOutputOperator.prop.topic</name>
+    <value>exactly-once</value>
+  </property>
+  <property>
+    <name>dt.application.KafkaExactlyOnce.operator.kafkaExactlyOnceOutputOperator.prop.properties(bootstrap.servers)</name>
+    <value>localhost:9092</value>
+  </property>
+  <property>
+    <name>dt.application.KafkaExactlyOnce.operator.kafkaExactlyOnceOutputOperator.prop.properties(key.serializer)</name>
+    <value>org.apache.kafka.common.serialization.StringSerializer</value>
```

Comment:
KafkaExactly once output operator doesn't require Key serializer and deserializer.